site/view.js

```diff
@@ -0,0 +1,150 @@
+'use strict';
+
+/**
+ * View class
+*/
+class View {
+  /**
+   * @constructor
+   * @param {Object.<string, HTMLElement>} elements
```

Comment:
`{Object<string, HTMLElement>}` is modern style.
A dot before type params are removed from Closure annotation rules.
https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#type-application